cli/pcluster/cfnconfig.py

```diff
@@ -657,36 +622,18 @@ def __init_efs_parameters(self):  # noqa: C901 FIXME!!!
                         elif key == "throughput_mode":
                             __throughput_mode = __temp__
                         # Separate sanity_check for fs_id, need to pass in fs_id and subnet_id
-                        if self.__sanity_check and self.__efs_options.get(key)[1] == "EFSFSId":
-                            __valid_mt = config_sanity.check_resource(
-                                self.region,
-                                self.aws_access_key_id,
-                                self.aws_secret_access_key,
-                                "EFSFSId",
-                                (__temp__, self.__master_subnet),
-                            )
-                        elif self.__sanity_check and self.__efs_options.get(key)[1] is not None:
-                            config_sanity.check_resource(
-                                self.region,
-                                self.aws_access_key_id,
-                                self.aws_secret_access_key,
-                                self.__efs_options.get(key)[1],
-                                __temp__,
-                            )
+                        if self.__efs_options.get(key)[1] == "EFSFSId":
+                            __valid_mt = self.__validate_resource("EFSFSId", (__temp__, self.__master_subnet))
+                        elif self.__efs_options.get(key)[1] is not None:
+                            self.__validate_resource(self.__efs_options.get(key)[1], __temp__)
                         __temp_efs_options.append(__temp__)
                     except configparser.NoOptionError:
                         __temp_efs_options.append("NONE")
                         pass
                 # Separate sanity_check for throughput settings,
                 # need to pass in throughput_mode and provisioned_throughput
-                if self.__sanity_check and (__provisioned_throughput is not None or __throughput_mode is not None):
-                    config_sanity.check_resource(
-                        self.region,
-                        self.aws_access_key_id,
-                        self.aws_secret_access_key,
-                        "EFSThroughput",
-                        (__throughput_mode, __provisioned_throughput),
-                    )
+                if __provisioned_throughput is not None or __throughput_mode is not None:
+                    self.__validate_resource("EFSThroughput", (__throughput_mode, __provisioned_throughput))
                 if __valid_mt:
```

Comment:
This could be not initialized

Reply:
Actually it was initialized to False before the try/catch block